jetstream.go

```diff
@@ -133,6 +139,31 @@ type AckOption func(opts *jsOpts) error
 // PublishOption configures publishing messages
 type PublishOption func(opts *jsOpts) error
 
+// SubscribeOption configures JetStream subscriber behaviour
+type SubscribeOption func(opts *jsOpts) error
+
+// SubscribeConsumer creates a JetStream consumer on stream based on DefaultConsumer and modified by opts
+func SubscribeConsumer(stream string, opts ...ConsumerConfigOption) SubscribeOption {
```

Comment:
Just Consumer I think..

Reply:
The Ack related ones start with Ack, publish ones with Publish etc. Makes them easy to find in IDE completion. I suspect we would add a good few more Subscribe related options in time